packages/api-gateway/src/drivers/http/routes/admin_panel/api/index.js

```diff
@@ -1,8 +1,10 @@
 const { adminPanelAdapters } = require('../../../adapters');
-// const schema = require('./schema');
+const schema = require('./schema');
 
 async function adminPanelRouter(fastify) {
-  await fastify.get('/', adminPanelAdapters.sayHello);
+
+	await fastify.get('/',adminPanelAdapters.sayHello);
```

Comment:
Can you replace the tab with 2 spaces and add a space after the comma? it is throwing eslint warnings
```suggestion
  await fastify.get('/', adminPanelAdapters.sayHello);
```